src/services/wallet-client/WalletClient.ts

```diff
@@ -26,6 +26,21 @@ class WalletClient {
     }
   }
 
+  async getCurrentWAudioBalance(): Promise<BNWei> {
+    try {
+      const balance = await AudiusBackend.getWAudioBalance()
+      return balance as BNWei
```

Comment:
BNWei?? 👀

Reply:
Yeah, it's `export type BNWei = Brand<BN, 'BNWei'>`
I think we did this so we know how many decimals we should be calculating with it. 
From libs, everything should be exposed in audio units with 18 decimals if with wei

Reply:
yeah I know why we did it, I was surprised that we were coercing Solana Waudio balances to Wei. But I guess it makes sense